pkg/dbstore/dbstore.go

```diff
@@ -42,18 +41,23 @@ func NewDBStore(lc fx.Lifecycle, config *config.Config) (*DB, error) {
 
 	p := path.Join(config.DataDir, "dashboard.sqlite.db")
 	log.Info("Dashboard initializing local storage file", zap.String("path", p))
-	gormDB, err := gorm.Open("sqlite3", p)
+	gormDB, err := gorm.Open(sqlite.Open(p), &gorm.Config{
+		Logger: zapgorm2.New(log.L()),
```

Comment:
nice catch! could you test whether it really works? (for example, you can simulate it by producing some error connections). In my current implementation the logger does not really work.

Reply:
I think it's works by check `cat /tmp/dashboard-data/dashboard.sqlite.db` output. And why the logger not work in current implementation?

Reply:
You can simulate a database query error, like introducing a syntax error. And then you will find unexpected log output in current implementation.